### PR TITLE
Improve Continuous Integration

### PR DESCRIPTION
**Pull request by MeekLogic**
PR for Issue #22

### Changes
- Removed .coveralls.yml as it shouldn't be needed
- Removed .npmrc as it shouldn't be needed (See Issue #22)
- Removed .travis.yml as it seems to not be used anymore
- Updated GitHub Workflow: Node CI
  - Upgraded [actions/checkout](https://github.com/actions/checkout) to `v2`
  - Moved Coveralls reporting to GitHub Workflow via [coverallsapp/github-action](https://github.com/coverallsapp/github-action)

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -1,27 +1,45 @@
...
+    branches: [ master ]
 
 jobs:
   build:
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node-version: [8.x, 10.x, 12.x, 13.x]
+        node-version: [10.x, 11.x, 12.x, 13.x]
```
[the remark this answers is not in the document]
Comment (MeekLogic) [reply .github/workflows/nodejs.yml]: I believe c8 requires >= 10

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -1,27 +1,45 @@
 name: Node CI
 
-on: [push, pull_request]
+on:
```
[the remark this answers is not in the document]
Comment (MeekLogic) [reply .github/workflows/nodejs.yml]: That's fine I'll switch it back

File: .github/workflows/nodejs.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (forzagreen) [reply .github/workflows/nodejs.yml]: All right. Then we can drop 8.x and previous versions.

**Comment by MeekLogic** [comment]
I've reverted Node CI run only on master push/pull.

Please take a look at #22 and consider the changes I have suggested for the NPM Publish CI workflow. This PR removes .npmrc.

File: .github/workflows/nodejs.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (MeekLogic) [reply .github/workflows/nodejs.yml]: Correction: c8 requires Node `>= 10.12`

**Pull request merged**